### PR TITLE
actual MLFlow docs

### PR DESCRIPTION
**Pull request by philipkiely-baseten**
* Example iPython notebook
* Creation doc

### CODE REVIEW

File: docs/create/mlflow.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Create a Truss of an MLFlow model
+
+[MLFlow](https://mlflow.org/) is a supported framework on Truss. To package a MLFlow model, follow the steps below or run [this Google Colab notebook](https://colab.research.google.com/github/basetenlabs/truss/blob/main/docs/notebooks/mlflow_example.ipynb).
```
Comment (amiruci) [inline docs/create/mlflow.md:3]: "_an_ MLFlow model" ?

**Review by amiruci — APPROVED** [review]

**Review by NishanthSingaraju — APPROVED** [review]

File: docs/create/mlflow.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### Create a Truss
+
+Truss uses MLFlow's [pyfunc](https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html) module in the packaging process. Once you have loaded the model, use the `mk_truss` command to package your model into a Truss.
+
+```python
+import os
+import truss
+
+path_to_model = f"mlruns/0/{MODEL_URI.split('/')[1]}/artifacts/model"
```
Comment (NishanthSingaraju) [inline docs/create/mlflow.md:44]: Why are you doing this instead of using the model_uri? Does the model uri not work?

**Review by NishanthSingaraju — APPROVED** [review]

File: docs/create/mlflow.md (hunk 2 above)
Comment (amiruci) [reply docs/create/mlflow.md:44]: yea let's simplify this please

Comment (philipkiely-baseten) [reply docs/create/mlflow.md]: Model uri totally works didn't need any of that parsing

**Pull request merged**